### PR TITLE
Fixes Tracking bugs

### LINKED ISSUE #16032: tracking_version_auto_create = true CREATE TABLE doesn't tracking changes on a new table
### Describe the bug

I would like to enable Change tracking feature for all operations.

I added the following config in `config.inc.php`

```php
// phpmyadmin tracking                                                                                                                
// https://docs.phpmyadmin.net/en/latest/config.html
$cfg['Servers'][$i]['tracking_version_auto_create'] = true;
$cfg['Servers'][$i]['tracking_default_statements'] = 'CREATE TABLE,ALTER TABLE,DROP TABLE,RENAME TABLE,CREATE INDEX,DROP INDEX,CREATE VIEW,ALTER VIEW,DROP VIEW,CREATE DATABASE,ALTER DATABASE,DROP DATABASE';
$cfg['Servers'][$i]['tracking_add_drop_view'] = true;
$cfg['Servers'][$i]['tracking_add_drop_table'] = true;
$cfg['Servers'][$i]['tracking_add_drop_database'] = true;
```

### To Reproduce

1. activate tracking in config.inc.php
2. login to phpmyadmin
3. go to a database
2. Click on 'Create table '
3. Fill up the form and create the table
4. The table doesn't appear in `pma_tracking` nor it has tracking enabled. 

### Expected behavior

The new table has tracking enabled by default and the `CREATE TABLE` is recorded.

### Server configuration

- Operating system: Debian 9 Stretch
- Web server: Apache/2.4.25 (Debian)
- Database version:  Server version: 10.1.41-MariaDB-0+deb9u1 - Debian 9.9 
- PHP version: Version PHP : 7.3.13-1+0~20191218.50+debian9~1.gbp23c2da
- phpMyAdmin version: phpMyAdmin-5.0.1-all-languages from download page.
    
If I put that at the end of config.inc.php I get:

```php
print_r($cfg);
die();
```
I have

```
Array
(
    [blowfish_secret] => Someblowfishsecret
    [TempDir] => /var/lib/phpmyadmin/tmp
    [LoginCookieValidity] => 86400
    [Servers] => Array
        (
            [1] => Array
                (
                    [auth_type] => cookie
                    [host] => localhost
                    [compress] => 
                    [AllowNoPassword] => 
                    [controlhost] => localhost
                    [controlport] => 
                    [controluser] => phpmyadmin
                    [controlpass] => pma_db_password_here
                    [pmadb] => phpmyadmin
                    [bookmarktable] => pma__bookmark
                    [relation] => pma__relation
                    [table_info] => pma__table_info
                    [table_coords] => pma__table_coords
                    [pdf_pages] => pma__pdf_pages
                    [column_info] => pma__column_info
                    [history] => pma__history
                    [table_uiprefs] => pma__table_uiprefs
                    [tracking] => pma__tracking
                    [userconfig] => pma__userconfig
                    [recent] => pma__recent
                    [favorite] => pma__favorite
                    [users] => pma__users
                    [usergroups] => pma__usergroups
                    [navigationhiding] => pma__navigationhiding
                    [savedsearches] => pma__savedsearches
                    [central_columns] => pma__central_columns
                    [designer_settings] => pma__designer_settings
                    [export_templates] => pma__export_templates
                    [tracking_version_auto_create] => 1
                    [tracking_default_statements] => CREATE TABLE,ALTER TABLE,DROP TABLE,RENAME TABLE,CREATE INDEX,DROP INDEX,CREATE VIEW,ALTER VIEW,DROP VIEW,CREATE DATABASE,ALTER DATABASE,DROP DATABASE
                    [tracking_add_drop_view] => 1
                    [tracking_add_drop_table] => 1
                    [tracking_add_drop_database] => 1
                )

        )

    [UploadDir] => 
    [SaveDir] => 
)
```

### Client configuration

- Browser: FireFox 74.0 (64-bit) ubuntu
- Operating system: xubuntu 18.04

### Additional context

phpmyadmin is configured from ansible playbook.

### PR DESCRIPTION
**Pull request by shucon**
### Description

Please describe your pull request.

Fixes: https://github.com/phpmyadmin/phpmyadmin/issues/16032 and another minor fix when clicking tracking icon next to table name

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by shucon** [comment]
@williamdes @ibennetch

File: libraries/classes/Tracking.php [hunk 1]
```diff
@@ -94,7 +94,7 @@ public function filter(
     /**
      * Function to get the list versions of the table
      */
-    public function getListOfVersionsOfTable(): array
+    public function getListOfVersionsOfTable()
```
Comment (williamdes) [inline libraries/classes/Tracking.php:97]: You can not make this change without finding the right type and adding a `@return` phpdoc or adding back a return typehint :)

File: libraries/classes/Tracking.php [hunk 2]
```diff
@@ -108,12 +108,11 @@ public function getListOfVersionsOfTable(): array
             ' ORDER BY version DESC ';
 
         $result = $relation->queryAsControlUser($sql_query);
-
-        if (! is_array($result)) {
-            return [];
+        if (get_class($result) == "mysqli_result") {
```
Comment (williamdes) [inline libraries/classes/Tracking.php:111]:
```suggestion
        if (get_class($result) === 'mysqli_result') {
```

**Comment by williamdes** [comment]
Could you make this one for `QA_5_0` without closing it ?

**Comment by shucon** [comment]
Closing in favour of https://github.com/phpmyadmin/phpmyadmin/pull/16389 . Minor fix when clicking tracking icon next to table name was already fixed in QA_5_0

**Pull request closed without merging**